windows/src/developer/TIKE/compile/CompileKeymanWeb.pas

```diff
@@ -1950,6 +1982,8 @@ function TCompileKeymanWeb.WriteCompiledKeyboard: string; {UTF8}
     'function %s()%s'+
     '{%s'+
     '%s%s%s'+
+    // Following line caches the Keyman major version
+    '%sthis._v=(typeof keyman!="undefined"&&typeof keyman.version=="string")?parseInt(keyman.version,10):9;%s'+
```

Comment:
Important note:  `this._v` is saved against the main keyboard object.

Reply:
Yes, and I am intending it for internal-use only (given it is a cache of a KeymanWeb property anyway)